.github/scripts/build.sh

```diff
@@ -52,6 +52,8 @@ ${DOCKER} bash -c "cd build && cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_DEBUG_WA
 echo =====
 ${DOCKER} bash -c "cd build && make -j $(nproc)"
 echo =====
+${DOCKER} bash -c "cd rust && cargo build"
```

Comment:
I would set the target-dir to `build/rust` so that everything built indirectly though cmake doesn't need to be compiled again.